src/test/java/br/com/alura/cibus/modelo/CozinhaTest.java

```diff
@@ -0,0 +1,33 @@
+package br.com.alura.cibus.modelo;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CozinhaTest {
+
+	@Test
+	void deveriaFalharCasoNomeEstiverVazio() {
+		Exception ex = assertThrows(Exception.class,() -> new Cozinha("Esse texto aqui deve conter ao menos 50 caracteres."));
```

Comment:
De acordo com as docs, as validações da classe `org.springframework.util.Assert` lançam `IllegalArgumentException`.

Seria legal fazer verificar que é essa exceção no assert. Faz mais sentido para validação de argumentos, né?
```suggestion
		assertThrows(IllegalArgumentException.class,() -> new Cozinha("Esse texto aqui deve conter ao menos 50 caracteres."));
```